### PR TITLE
Bookstore: use redux in components 

### PR DESCRIPTION
**Pull request by ogagaoghene**
# Bookstore: use redux in components

## Description
Use redux in components to implement add and remove functionality

### CODE REVIEW

File: src/components/Categories.jsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    <div className="categories">
+      <h1>Categories list</h1>
+      <button
+        type="button"
+        onClick={() => {
+          dispatch(checkStatus());
+        }}
+      >
+        Check Status
+      </button>
```
Comment (gemmen29) [inline src/components/Categories.jsx]: - [x] Good job on dispatching `checkStatus` action but Please make sure that when the user clicks the `Check Status` button, the text `Under Construction` appear on the page.

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @ogagaoghene, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Categories.jsx (hunk 1 above)
Comment (ogagaoghene) [reply src/components/Categories.jsx]: I have implemented this functionality as requested.

**Review by sumairq — APPROVED** [review]
### Hi @ogagaoghene  , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Git flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ogagaoghene** [comment]
Thanks for your swift approval. I appreciate you greatly.

**Pull request merged**